sim/midas/src/main/scala/midas/platform/ZynqShim.scala

```diff
@@ -7,23 +7,26 @@ import chisel3._
 import chisel3.util._
 import junctions._
 import freechips.rocketchip.config.{Parameters, Field}
+import freechips.rocketchip.diplomacy.{LazyModule}
 import freechips.rocketchip.util.ParameterizedBundle
 
-import midas.core.{HostMemChannelNastiKey, ChannelWidth}
+import midas.core.ChannelWidth
 
-abstract class PlatformShim(implicit p: Parameters) extends Module {
-  def top: midas.core.FPGATop
+abstract class PlatformShim(implicit p: Parameters) extends MultiIOModule {
```

Comment:
I feel like this class should be in a separate Scala file.

Reply:
Agreed.